.github/workflows/release_nightly_on_schedule.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build-n-publish:
-    if: github.event_name == 'workflow_dispatch' || github.repository == 'hpcaitech/ColossalAI'
+    if: github.event_name == 'workflow_dispatch' && github.repository == 'hpcaitech/ColossalAI'
```

Comment:
This is not right. You can just keep this:

```
if: github.repository == 'hpcaitech/ColossalAI'
```

Reply:
ok, I found I misunderstand workflow_dispatch.
And DONE

Reply:
I mean, you can remove the `github.event_name == 'workflow_dispatch'`, and leave it like
`if: github.repository == 'hpcaitech/ColossalAI'`